website/src/css/_navbar.scss

```diff
@@ -22,3 +22,47 @@
   padding: 5px;
   font-weight: 300;
 }
+/*===========*/
+
+/* Fix navbar logo */
+.navbar__logo {
+  height: 26px;
+  margin: 0 10px 0 8px;
+}
+
+/* Restyle gamburger button */
+
+.navbar {
+  position: relative;
+  height: 70px;
+}
+
+.navbar__toggle {
+  width: 40px;
+  height: 40px;
+  background-color: var(--detox-color-a3);
+  border-radius: 10px;
+  justify-content: center;
+  align-items: center;
+  position: absolute;
+  margin-right: 0;
+  top: 15px;
+  right: 25px;
```

Comment:
Actually I'm not sure that the hamburger should move to the right in compact mode. 

I'd rather put it under media query 768px.

On the other hand, we don't have specific designs for that intermediate resolution so it's worth checking with our designers.

<img width="792" alt="image" src="https://user-images.githubusercontent.com/1962469/218152519-268b3d76-1a0c-495d-94df-6971aea1b67a.png">

Reply:
I will fix it according to your suggestion.

Reply:
Cool.